config/config.txt

```diff
@@ -46,6 +46,14 @@ ADMIN_LEGACY_SYSTEM
 ##If enabled, each time admins are loaded ranks the database will be updated with the current ranks and their flags
 #LOAD_LEGACY_RANKS_ONLY
 
+## Comment this out if you want to use the SQL based mentor system, the legacy system uses mentors.txt.
```

Comment:
Note to self: Add this on server box

Reply:
You'll also want to run the included SQL file to add the database tables.